### ISSUE TITLE
Pycharm debugger crashes on pytest as soon as nameko package is installed 

### ISSUE THREAD

**Issue by wasmitnetzen** [nameko/nameko#588]
(Note: I think it is more likely to be a PyCharm bug, which is why I have also created an issue on their [bug tracker](https://youtrack.jetbrains.com/issue/PY-32611).)
When I try to run a simple pytest in PyCharm in debug mode when nameko is installed, the debugger crashes with a SIGABRT signal.

**To Reproduce**
* Make a PyCharm project
* Install nameko
* Make a test directory
* Make a test file
```
def test_nameko():
    assert True
    print("Hello World!")
```

Run the PyCharm debugger on this file.

**Expected behavior**
A normal output, as can be seen when running the file normally.

**Environment:**
 - Nameko version: 2.11.0
 - Python version: 3.6.7
 - OS: Ubuntu 18.10

There is also #560 which might have been the same problem, but there is no helpful content there.

**Comment by mattbennett** [comment]
It seems like this is caused by the eventlet monkey patch. Not easy to fix I'm afraid.

There's a related stack overflow post here: https://stackoverflow.com/questions/38448305/can-not-remote-debug-if-there-is-eventlet-monkey-patch-in-code

**Comment by daimon99** [comment]
how about comment this code in testing/pytest.py?

```
def pytest_load_initial_conftests():
    # make sure we monkey_patch before local conftests
    # import eventlet
    # eventlet.monkey_patch()
    pass
```

**Comment by mattbennett** [comment]
@daimon99 that code applies the monkey patch that enables eventlet's cooperative yielding.

Pycharm would stop crashing, but your Nameko service wouldn't run anymore ;)

Going to close this issue because there's nothing we can do in Nameko to mitigate it.

**Comment by no-life** [comment]
macOS Catalina, python 3.7, PyCharm 2019.3.1.
Setting "Preferences > Python Debugger: Gevent compatible" solved problem for me

**Comment by daimon99** [comment]
It is not fixed In PyCharm 2019.2 version which I am using:(
It is a good news that jetbrains fix this bug！

> macOS Catalina, python 3.7, PyCharm 2019.3.1.
> Setting "Preferences > Python Debugger: Gevent compatible" solved problem for me
